### PR TITLE
Changes in Fixbook class 

### PR DESCRIPTION
**Pull request by BLakshmiPrasanna**
…elines

### CODE REVIEW

**Review by abhimaniac4 — APPROVED** [review]

File: library/fixbook/FixBookControl.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	private Book currentBook;
 
 
-	public fixbookControl() {
+	public FixBookControl() {
```
Comment (iamharisali) [inline library/fixbook/FixBookControl.java]: it should be fixBookControl(). The first letter should not be capital

File: library/fixbook/FixBookUI.java [hunk 2]
```diff
@@ -48,9 +48,9 @@ public void runFixBookMethods() {
 				break;	
 				
 			case FIXING:
-				String AnS = input("Fix Book? (Y/N) : ");
+				String anSwer = input("Fix Book? (Y/N) : ");
```
Comment (iamharisali) [inline library/fixbook/FixBookUI.java]: "anSwer" should be "answer"

File: library/fixbook/FixBookControl.java (hunk 1 above)
Comment (abhimaniac4) [reply library/fixbook/FixBookControl.java]: @BLakshmiPrasanna Rework needed for particular error raised by reviewer Haris Ali

File: library/fixbook/FixBookUI.java (hunk 2 above)
Comment (abhimaniac4) [reply library/fixbook/FixBookUI.java]: @BLakshmiPrasanna Rework needed for particular error raised by Haris Ali

File: library/fixbook/FixBookControl.java (hunk 1 above)
Comment (BLakshmiPrasanna) [reply library/fixbook/FixBookControl.java]: @abhimaniac4 Fixed the issue please review

File: library/fixbook/FixBookUI.java (hunk 2 above)
Comment (BLakshmiPrasanna) [reply library/fixbook/FixBookUI.java]: @abhimaniac4 Already changed this issue in commit [d0a3ff2] please review

**Review by iamharisali — APPROVED** [review]

**Review by abhimaniac4 — COMMENTED** [review]
Changes made correctly.

**Pull request merged**